util/util.go

```diff
@@ -183,3 +183,51 @@ func FormatOutput(in interface{}, tmplt string) (string, error) {
 
 	return out.String(), nil
 }
+
+func CheckoutPR() {
```

Comment:
This doesn't ever seem to be called - additionally I'm a bit concerned about the duplication of the shell script below. Can it be unified with the one over in commands/copr.go